fit-checker/utils/change-detect/entity/ChangeType.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//  fit-checker
+//
+//  Created by Jakub Tucek on 21/01/17.
+//  Copyright © 2017 Josef Dolezal. All rights reserved.
+//
+
+import Foundation
+
+
+/// Enum defining change type
+///
+/// - removed: item was removed in newe data
+/// - added: item was added in new data
+/// - modified: item was modified in new data
+enum ChangeType {
```

Comment:
Same as `Change.swift` comment. Because swift does not use namespaces inside module, this enum is visible in whole app.